index.html

```diff
@@ -1,223 +1,271 @@
+              <p class="numbers">04</p>
+              <img class="image-technology" src="/images/csv.svg" alt="LOGO CSV">
+              <h3 class="title-small">CSV Download</h3>
+              <p>All your information is stored on secure cloud servers</p>
+            </li>
+          </ul>
+        </div>
+      </section>
+      <section class="margin-70">
+        <div class="container">
+          <div class="row">
+            <div class="col col-5">
+              <h2 class="title-medium">Stay focused on saving money</h2>
+              <p class="text-features">It is important to stay focused on saving money in any way you can. We help you monitor your spending habits so you can easily spot and cut any unnecessary expenses. Simply join today to get started!</p>
+              <a href="">
+                <img src="/images/playstore.svg" alt="Logo Playstore">
+              </a>
+              <a href="">
+                <img src="/images/appstore.svg" alt="Logo Appstore">
+              </a>
+              <a href="">
+                <img src="/images/windows.svg" alt="Logo Windows">
+              </a>
+            </div>
+            <div class="col col-6 offset-1">
+              <iframe class="main-map" src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d1322.7878891344503!2d35.053136366149914!3d48.464668530716736!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x40dbe2c338d87c15%3A0x8016e636e75a7c39!2z0L_Quy4g0KPRgdC_0LXQvdGB0LrQsNGPLCDQlNC90LjQv9GA0L4sINCU0L3QtdC_0YDQvtC_0LXRgtGA0L7QstGB0LrQsNGPINC-0LHQu9Cw0YHRgtGMLCA0OTAwMA!5e0!3m2!1sru!2sua!4v1606566116999!5m2!1sru!2sua" width="600" height="450" style="border:0;" allowfullscreen="" aria-hidden="false" tabindex="0"></iframe>
+            </div>
+          </div>
+        </div>
+      </section>
+      <section class="contact margin-70">
+        <div class="container">
+          <div class="row">
+            <div class="col col-5">
+              <h2 class="title-medium">Contact</h2>
+              <p class="ptext">Questions or concerns? Just fill out the form below and our support team will get back to you within 24 hours. But I must explain to you how all this mistaken idea of denouncing pleasure and praising pain was born and I will give you a complete account of the system, and expound the actual teachings of the great explorer of the truth, the master-builder of human happiness.</p>
+            </div>
+            <div class="col col-6 offset-1">
+              <form>
+                <div class="row">
+                    <div class="col col-6">
+                      <label>
+                      <input class="select-main"  type="text" name="user_first_name" placeholder="First Name" required>
+                      </label>
+                    </div>
+                    <div class="col col-6">
+                      <label>
+                        <input class="select-main" type="text" name="user_last_name" placeholder="Last Name" required>

```

Comment:
Did you you project with lighthouse? I guess there several a11y issues. Check it please and fix possible issues.